src/Proto.Cluster/Identity/IdentityStorageLookup.cs

```diff
@@ -36,7 +37,7 @@ public async Task SetupAsync(Cluster cluster, string[] kinds, bool isClient)
         await Storage.Init();
 
         var workerProps = Props.FromProducer(() => new IdentityStorageWorker(this));
-        _worker = _system.Root.Spawn(workerProps);
+        _worker = _system.Root.SpawnNamedSystem(workerProps, WorkerActorName);
```

Comment:
I'm not sure about whether this should be a system actor. As far as I understand this worker lives alongside the IdentityStorageLookup and stops when that is stopped hence in that case it makes sense for me that this is spawn as a system actor.